src/styles/colors.js

```diff
@@ -17,7 +17,7 @@ const base = {
   lightestGrey: '#E9EBEF', // '238, 233, 232'
   lightGrey: '#CDCFD4', // '205, 207, 212'
   mediumGrey: '#A1A5B3', // '161, 165, 179'
-  orange: '#FFB200', // '255, 178, 0'
+  orange: '#FF9900', // '255, 178, 0'
```

Comment:
You might wanna update the comment since the color has changed
```suggestion
  orange: '#FF9900', // '255, 153, 0'
```